transactions/src/main/java/org/jboss/as/txn/subsystem/TransactionSubsystemRootResourceDefinition.java

```diff
@@ -521,6 +521,8 @@ protected boolean applyUpdateToRuntime(final OperationContext context, final Mod
             throws OperationFailedException {
             int timeout = resolvedValue.asInt();
 
+            // TxControl allow timeout to be set to 0 with meaning of no timeout at all
+            arjPropertyManager.getCoordinatorEnvironmentBean().setDefaultTimeout(timeout);
```

Comment:
Note that there is a revertUpdateToRuntime operation too

Reply:
yes, correct. thank you!